.github/workflows/unitTests.yaml

```diff
@@ -30,15 +30,17 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Checkout code
-      uses: actions/checkout@v3
+      uses: actions/checkout@v4
     - name: Setup Environment
       uses: ./.github/actions/setupEnvironment
     - run: yarn run unit-ci
-    - uses: actions/upload-artifact@v1
```

Comment:
This `upload-artifact` is old debugging code that hasn't been used for a long time

Reply:
Hmm - that's fair. I removed it before as it was interfering with my previous attempt at getting this working, but I'm using a different method now where that isn't true anymore. I'll put this upload back in.